__test__/Gallery.test.js

```diff
@@ -20,37 +20,48 @@ describe('Gallery', () => {
     wrapper = mount(
       <Gallery photos={photos} onClick={handleClick}/>
     );
-    wrapper.setState({containerWidth: '1139'})
+    wrapper.setState({containerWidth: 1139})
     expect(wrapper).toMatchSnapshot();
   });
 
   it('renders correctly if there are more columns than photos', () => {
     wrapper = mount(
       <Gallery photos={photos} columns={10} onClick={handleClick}/>
     );
-    wrapper.setState({containerWidth: '1139'})
+    wrapper.setState({containerWidth: 1139})
     expect(wrapper).toMatchSnapshot();
   });
+
   it('renders correctly after click', () => {
     wrapper = mount(
       <Gallery photos={photos} onClick={handleClick} />
     );
-    wrapper.setState({containerWidth: '1139'})
+    wrapper.setState({containerWidth: 1139})
     wrapper.find('img').first().simulate('click');
     expect(wrapper).toMatchSnapshot();
   });
 
   it('renders correctly with direction set to column', () => {
-     wrapper = mount(
+    wrapper = mount(
       <Gallery photos={photos} onClick={handleClick} direction={'column'}/>
     );
-    wrapper.setState({containerWidth: '1139'})
+    wrapper.setState({containerWidth: 1139})
```

Comment:
Noticed these `setStates` were setting the width as a string, which doesn't seem correct, so i've updated them.